### PR TITLE
Large refactor: removes Lerna & renames to Ignite CLI

### PR DESCRIPTION
**Pull request by jamonholmgren**
• Moves DevScreens to [new repo](https://github.com/infinitered/ignite-dev-screens)
• Folds `ignite-integration-tests` into `ignite-cli` package
• Moves `ignite-cli` package into main folder
• Removes Lerna entirely
• Renames `Ignite` and `Ignite 2` to `Ignite CLI` everywhere

_Many_ thanks to @robinheinze for pairing with me on this.

The only code changes I did were changing paths and renaming Ignite to Ignite CLI. The rest are shuffling files around and deleting stuff.

### CODE REVIEW

**Comment by jamonholmgren** [comment]
CI tests are failing because we'll need to modify them for this new structure. They pass locally.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 
-#  Contributing to Ignite
+#  Contributing to Ignite CLI
```
Comment (robinheinze) [inline .github/CONTRIBUTING.md]: Should we change Ignite 2 to Ignite CLI on the website as well? https://infinite.red/ignite

File: src/extensions/ignite.js [hunk 2]
```diff
@@ -8,14 +8,14 @@ const findIgnitePluginsExt = require('./ignite/findIgnitePlugins')
 const addModuleExt = require('./ignite/addModule')
 const addAndroidPermissionExt = require('./ignite/addAndroidPermission')
 const removeModuleExt = require('./ignite/removeModule')
-const addScreenExamplesExt = require('./ignite/addScreenExamples') // Deprecated 3/2/17, Ignite 2 Beta
+const addScreenExamplesExt = require('./ignite/addScreenExamples') // Deprecated 3/2/17, Ignite CLI Beta
```
Comment (derekgreenberg) [inline src/extensions/ignite.js]: There's a bunch of Deprecated 3/2/17 comments here. Were these require statements deprecated on 3/2/17?

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (jamonholmgren) [inline .github/CONTRIBUTING.md]: @robinheinze Good idea. Tracking here: https://github.com/infinitered/ir_website/issues/326

File: src/extensions/ignite.js (hunk 2 above)
Comment (jamonholmgren) [inline src/extensions/ignite.js]: @robinheinze added these -- we should probably keep them in until 2.1+.

**Comment by GantMan** [comment]
Hey, what happened to the testRelease CI?  It looks deleted in this

File: bin/testRelease [hunk 3]
```diff
@@ -1,145 +0,0 @@
...
-echo '#                  ._______.                            #'
-echo '#                   | \   / |                           #'
-echo '#                .--|.O.|.O.|______.                    #'
-echo '#              __).-| = | = |/   \ |                    #'
-echo "#              >__) (.'---\`.)Q.|.Q.|--.                 #"
-echo '#                    \\___// = | = |-.(__               #'
-echo "#                     \`---'( .---. ) (__<               #"
-echo "#                           \\\\.-.//                     #"
-echo "#                            \`---'                      #"
-echo '#########################################################'
```
Comment (GantMan) [inline bin/testRelease]: This CI test is critical.  Can you identify where you moved it to?  I don't see it renamed... just deleted?

Comment (jamonholmgren) [inline bin/testRelease]: Re-introduced in #1011 @GantMan. It seemed to duplicate what we already have in CI, but there are some additional steps in it, so it's worth keeping.

**Pull request merged**